src/main/java/randoop/condition/PreconditionOutcomeTable.java

```diff
@@ -0,0 +1,138 @@
+package randoop.condition;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import randoop.test.ExpectedExceptionGenerator;
+import randoop.test.ExtendGenerator;
+import randoop.test.InvalidCheckGenerator;
+import randoop.test.PostConditionCheckGenerator;
+import randoop.test.TestCheckGenerator;
+
+/**
+ * Records the outcome of checking all of the preconditions for a method. Each table entry records:
```

Comment:
There was only one line of documentation for this class.  There wasn't even a cross-reference to where documentation could be found.  Please document each class.  Do avoid duplication, but it's better to cross-reference from elsewhere to class documentation than to put (partial) class documentation elsewhere where it may not be discovered.